python/test/test_polars.py

```diff
@@ -48,16 +48,20 @@ def example_lf(margin=None, **kwargs):
     domains, series = example_series()
     lf_domain, lf = dp.lazyframe_domain(domains), pl.LazyFrame(series)
     if margin is not None:
+        margin = [col if isinstance(col, pl.Expr) else pl.col(col) for col in margin]
         lf_domain = dp.with_margin(lf_domain, by=margin, **kwargs)
     return lf_domain, lf
 
 
 def test_expr_domain():
     series_domains, _ = example_series()
     dp.wild_expr_domain(series_domains)
-    dp.wild_expr_domain(series_domains, by=["A", "B"])
-    dp.wild_expr_domain(series_domains, by=["A", "B"], max_num_partitions=10)
-    dp.wild_expr_domain(series_domains, by=["A", "B"], public_info="keys")
```

Comment:
Would it be good to retain the old test to make sure it continues to work as well?